examples/waffle/test/erc20.spec.js

```diff
@@ -41,12 +36,7 @@ describe('ERC20 smart contract', () => {
 
   describe('when using a deployed contract instance', () => {
     before(async () => {
-      wallet = await provider.getSigner(0)
```

Comment:
cc @platocrat we shouldn't encourage folks to be using a built-in signer in their Geth node

Reply:
That was my change, good point.